### PR TITLE
Add LICENSE file

### PR DESCRIPTION
**Pull request by gbraad**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (gbraad) [inline LICENSE:189]: This isn't filled.

**Review by gbraad — COMMENTED** [review]
Missing owner

File: LICENSE (hunk 1 above)
Comment (anjannath) [reply LICENSE:189]: Please fill the required information, otherwise LGTM

Comment (cfergeau) [reply LICENSE:189]: Is this something to keep in the LICENSE file? or should we end the LICENSE file just before `END OF TERMS AND CONDITIONS`, and add this boilerplate headers to the source files?

Comment (gbraad) [reply LICENSE:189]: I hate the practice to keep an eye on the source templates, but it does make sense.

Comment (gbraad) [reply LICENSE:189]: BTW, @anjannath what do we do for the macOS files?

Comment (anjannath) [reply LICENSE:189]: We are not adding this license text in each source files, also for the crc repo. Only the LICENSE file is added

Comment (cfergeau) [reply LICENSE:189]: https://infra.apache.org/licensing-howto.html#step-by-step has the instructions, my understanding is that we copy the `LICENSE`  file as is, and that we need to add a `NOTICE` file with the tray-specific conditions (copyright, authors, ...)

Comment (anjannath) [reply LICENSE:189]: @gbraad can you please add the `NOTICE` file in this PR itself? we do need to follow the same for the other repos

Comment (anjannath) [reply LICENSE:189]: Will create a follow up issue to add the NOTICE file for now having the LICENSE is already better than before

**Pull request merged**